src/internal/keycode.ts

```diff
@@ -17,8 +17,4 @@ export enum KeyCode {
   escape = 27,
   enter = 13,
   tab = 9,
-  shift = 16,
-  control = 17,
-  alt = 18,
-  meta = 91,
```

Comment:
Never used enum members. Actually, we should cease the entire `KeyCode` enum and adopt `event.key` instead.

--
P.S. Enum members do not get minified, so it makes sense to remove unused code here just for this reason